op-e2e/e2eutils/disputegame/helper.go

```diff
@@ -141,6 +143,46 @@ func (h *FactoryHelper) StartAlphabetGame(ctx context.Context, claimedAlphabet s
 	}
 }
 
+func (h *FactoryHelper) StartOutputCannonGame(ctx context.Context, rollupEndpoint string, rootClaim common.Hash) *OutputCannonGameHelper {
+	l2BlockNumber := h.waitForProposals(ctx)
+	l1Head := h.checkpointL1Block(ctx)
```

Comment:
I think we can also just reuse the refactored `prepareCannonGame` method here like so
```suggestion
	l2BlockNumber, l1Head := h.prepareCannonGame(ctx)
```